hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -50,24 +49,24 @@ spec = parallel $
       validateRunOptions (defaultRunOptions{hydraVerificationKeys = hydraKeys, chainConfig = chainCfg})
         `shouldBe` Left CardanoAndHydraKeysMissmatch
 
-    it "parses with default node-id set" $
-      setFlags [] `shouldParse` Run defaultRunOptions
+    it "parses with node-id and testnet-magic set" $
+      defaultNodeOptions `shouldParse` Run defaultRunOptions
 
     it "parses --host option given valid IPv4 and IPv6 addresses" $ do
-      setFlags ["--host", "127.0.0.1"]
+      (defaultNodeOptions <> ["--host", "127.0.0.1"])
```

Comment:
The previous interface was nicer, as we did not had to add parenthesis and `<>`.

Should: keep `setFlags` function.